snap/kubefed.yaml

```diff
@@ -0,0 +1,20 @@
+name: kubefed
+version: 'KUBE_VERSION'
+summary: kubefed controls the Kubernetes cluster federation manager.
+description: |
+  kubefed is a command line interface for running commands against Kubernetes
+  federated clusters.
+grade: stable
+confinement: classic
```

Comment:
Did some tests with kubefed on stricted confinement and the user experience is not good.

Kubefed is a cli client that needs to (among others) read and write into .kube/config. The home plug of snaps has the limitation of not being able to access hidden files. Keeping us aligned with best practices means we would better use classic confinement.